setup.cfg

```diff
@@ -2,6 +2,11 @@
 name = aiomysql
 url = https://github.com/aio-libs/aiomysql
 download_url = https://pypi.python.org/pypi/aiomysql
+project_urls =
+  CI: GitHub = https://github.com/aio-libs/aiomysql/actions
+  Docs: RTD = https://aiomysql.readthedocs.io/
+  GitHub: issues = https://github.com/aio-libs/aiomysql/issues
```

Comment:
Also, maybe link https://github.com/aio-libs/aiomysql/discussions too

Reply:
done in #739